src/npcmove.cpp

```diff
@@ -2632,13 +2582,23 @@ bool npc::complain()
 
     // When tired, complain every 30 minutes
     // If massively tired, ignore restrictions
-    if( complaints[fatigue_string] < calendar::turn - MINUTES(30) &&
+    if( fatigue > TIRED &&
+        complaints[fatigue_string] < calendar::turn - MINUTES(30) &&
         (rules.allow_complain || fatigue > MASSIVE_FATIGUE - 100) ) {
         say( "<yawn>" );
         complaints[fatigue_string] = calendar::turn;
         return true;
     }
 
+    // Radiation every 10 minutes
+    if( radiation > 90 &&
+        complaints[radiation_string] < calendar::turn - MINUTES(10) &&
+        (rules.allow_complain || radiation > 150) ) {
+        say( _("I'm suffering from radiation sickness...") );
```

Comment:
There's a lot of room for this to be a gruesome list of symptoms rather than a simple observation.